app/controllers/casa_admins_controller.rb

```diff
@@ -35,14 +35,13 @@ def set_admin
   end
 
   def update_casa_admin_params
-    CasaAdminParameters.new(params)
-  end
-
-  def casa_admin_values
-    { password: SecureRandom.hex(10), casa_org_id: current_user.casa_org_id }
+    CasaAdminParameters.new(params).with_only(:email, :display_name)
   end
 
-  def casa_admin_params
-    params.require(:casa_admin).permit(:display_name, :email, :password, :casa_org_id)
+  def create_casa_admin_params
+    CasaAdminParameters.new(params)
```

Comment:
@jmkoni can I get your eyes on this change as well? I noticed this `*Parameters` pattern being used in other places and figured that would fi this scenario nicely.

Is this how we'd like to handle strong params moving forward?

cc @derricklannaman, @stevelandwehr.

Reply:
@drborges Is this way still whitelisting the acceptable incoming params from the form?

Reply:
@derricklannaman yes. I also ran a quick smoke test to make sure the create functionality still works in addition to the system spec passing 👍